stand/efi/loader/Makefile

```diff
@@ -81,7 +81,7 @@ CFLAGS+=	-DEFI_STAGING_SIZE=${EFI_STAGING_SIZE}
 CFLAGS+= -DEFI_SECUREBOOT
 .endif
 
-NEWVERSWHAT=	"EFI loader" ${MACHINE}
+NEWVERSWHAT?=	"EFI loader" ${MACHINE}
```

Comment:
this one chunk can be done in a commit with a good commit message about allowing it to be overriden. It's otherwise ready to land.